features/loadods.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Load ods files
+  I want to load ods files into databaker
+
+  Scenario: Load ods file 1
+    Given we load a file named "2018internationaltradeinservicesdatatables.ods"
+    Then we confirm the names of the loaded tabs are equal to:
+    """
+    ["Notes", "Index", "1. NUTS1, industry", "2. NUTS1, industry, destination",
+    "3. NUTS2, industry", "4. NUTS2, industry, destination", "5. NUTS3, destination",
+    "6. City Region, industry", "7. City Region, industry, dest.", "8. Travel", "9. Tidy format"]
+    """
+    And the output "some_name" should be equal to:
```

Comment:
this cant be right 😆 , needs a look but I'm guessing that variable is never used. Should probably clean up whatever this is (I'm betting a leftover from my initial hacking).

Reply:
Reworded this step and removed the variable.